test/subgraph-tests/position.test.ts

```
@@ -44,12 +44,25 @@ describe('Position fetching logic from subgraph', () => {
   });
-  it.only('should return only closed positions of this timeframe', async () => {
+  it('should return only closed positions of this timeframe', async () => {
     const parifiSdk = await getParifiSdkInstanceForTesting();
     const startTime = 1743710952;
     const endTime = 1743763152;
     const positionData = await parifiSdk.subgraph.getAllPositionHistoryWithTime(startTime, endTime);
     console.log('Position data:::', positionData);
     expect(positionData.length).toEqual(6);
   });
+
+  it.only('should return positions for a snx account id', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)